azure-pipelines.yml

```diff
@@ -19,7 +19,12 @@ variables:
     _HelixSource: official/aspnet/AspNetCore-Tooling/$(Build.SourceBranch)
     _SignType: real
     _SignArgs: /p:DotNetSignType=$(_SignType) /p:TeamName=$(_TeamName)
-    _OfficialBuildIdArgs: /p:OfficialBuildId=$(Build.BuildNumber)
+    _OfficialBuildIdArgs: /p:OfficialBuildId=$(Build.BuildNumber) 
+      /p:ManifestTeamProject=AspNet
+      /p:ManifestTeamName=AspNetCore-Tooling
+      /p:ManifestBuildBranch=$(Build.SourceBranch)
+      /p:ManifestBuildNumber=$(Build.BuildNumber)
+      /p:ManifestPublishUrl=https://vsdrop.corp.microsoft.com/file/v1/Products/AspNet/AspNetCore-Tooling/$(Build.SourceBranch)/$(Build.BuildNumber);
```

Comment:
Yeah it is. It felt wierd to this this define in an MSBuild file. 

It should probably be a variable defined outside of the YAML to be totally honest, but I'm not sure how that works. 

I'm not worried about this being super clean right now because it's a workaround, and I need to trigger an official build to even see if it fixes my problem.